src/katello/deb_tracer.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import absolute_import
+from os import path
+from shutil import which
+import subprocess
+
+# The file is created by /etc/kernel/postinst.d/unattended-upgrades (part of unattended-upgrades pkg)
+REBOOT_NEEDED_FLAG = "/var/run/reboot-required"
+
+
+class AptTracerApp:
+    pass
+
+
+def check_for_needrestart():
+    return which("needrestart") is not None
+
+
+def use_flag():
+    apps = []
+    if path.isfile(REBOOT_NEEDED_FLAG):
+        app = AptTracerApp()
+        app.name = "kernel"
+        app.helper = "You will have to reboot your computer"
+        app.type = "static"
+        apps.append(app)
+    return apps
+
+
+def needrestart():
+    process = subprocess.Popen(["needrestart", "-b"], stdout=subprocess.PIPE)
+    output = process.communicate()
+    return output[0].decode().split("\n")
+
+
+def use_needrestart():
+    apps = []
+    services = []
+    kasta = None
```

Comment:
What is `kasta`? later on you use `ksta`.

Reply:
I defined it to use as ksta. It is a mistake, because we do not need it.